test/jasmine/tests/annotations_test.js

```diff
@@ -554,9 +554,9 @@ describe('annotations autorange', function() {
     it('should adapt to relayout calls', function(done) {
         Plotly.plot(gd, mock).then(function() {
             assertRanges(
-                [0.97, 2.03], [0.97, 2.03],
-                ['2000-10-01 08:23:18.0583', '2001-06-05 19:20:23.301'], [-0.245, 4.245],
-                [0.9, 2.1], [0.86, 2.14]
+                [0.91, 2.09], [0.91, 2.09],
```

Comment:
> so these values changed because the `annotations-autorange` mock changed?

correct. I guess that's a hazard of reusing mocks for both images and jasmine tests...